lib/upfluence/endpoint/api_endpoint.rb

```diff
@@ -38,6 +39,18 @@ def respond_with(resource, *args)
           halt [status, result] || 404
         end
       end
+
+      def json_params
+        begin
+          JSON.parse(request.body.read).with_indifferent_access
```

Comment:
I think you will need to require 'active_support/core_ext/hash' before using the function